recipes/liburing/all/conanfile.py

```diff
@@ -13,7 +13,7 @@ class Liburing(ConanFile):
     name = "liburing"
     description = ("helpers to setup and teardown io_uring instances, and also a simplified interface for "
                    "applications that don't need (or want) to deal with the full kernel side implementation.")
-    license = "GPL-2.0-or-later"
+    license = ("GPL-2.0-or-later", "MIT")
```

Comment:
```suggestion
    license = ("GPL-2.0-or-later", "MIT", "LGPL-2.1-or-later")
```

https://github.com/axboe/liburing/blob/master/COPYING

Present already in 2.8